modules/tensor_mechanics/test/tests/ad_anisotropic_plasticity/ad_aniso_plasticity_y.i

```diff
@@ -0,0 +1,272 @@
+[Mesh]
+  [gen]
+    type = GeneratedMeshGenerator
+    dim = 3
+
+    # Original verification nx = 2, ny = 10, nz = 2
+    nx = 2
+    ny = 10
+    nz = 2
+
+    xmin = 0.0
+    ymin = 0.0
+    zmin = 0.0
+
+    xmax = 1.0
+    ymax = 10.0
+    zmax = 1.0
+  []
+  [corner_node]
+    type = ExtraNodesetGenerator
+    new_boundary = '100'
+    nodes = '4 10'
+    input = gen
+  []
+  [corner_node_2]
+    type = ExtraNodesetGenerator
+    new_boundary = '101'
+    nodes = '1 67'
+    input = corner_node
+  []
+[]
+
+[GlobalParams]
+  displacements = 'disp_x disp_y disp_z'
+  volumetric_locking_correction = true
+  order = FIRST
+  family = LAGRANGE
+[]
+
+[AuxVariables]
+  [hydrostatic_stress]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [plasticity_strain_xx]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [plasticity_strain_xy]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [plasticity_strain_yy]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [stress_yy]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [elastic_strain_yy]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+[]
+[Variables]
+  [disp_x]
+  []
+  [disp_y]
+  []
+  [disp_z]
+  []
+[]
+[AuxKernels]
+  [hydrostatic_stress]
+    type = ADRankTwoScalarAux
+    variable = hydrostatic_stress
+    rank_two_tensor = stress
+    scalar_type = Hydrostatic
+  []
+  [plasticity_strain_xx]
+    type = ADRankTwoAux
+    rank_two_tensor = trial_plasticity_plasticity_strain
+    variable = plasticity_strain_xx
+    index_i = 0
+    index_j = 0
+  []
+  [plasticity_strain_xy]
+    type = ADRankTwoAux
+    rank_two_tensor = trial_plasticity_plasticity_strain
+    variable = plasticity_strain_xy
+    index_i = 0
+    index_j = 1
+  []
+  [plasticity_strain_yy]
+    type = ADRankTwoAux
+    rank_two_tensor = trial_plasticity_plasticity_strain
+    variable = plasticity_strain_yy
```

Comment:
we have `plastic_strain_??` as a standard material property. The two names here are a bit confusing.

Reply:
and any reason you removed the one `add_variables` line rather than the variables block above?